Pod/Classes/ObjC/SKTypeDefs.h

```diff
@@ -12,4 +12,14 @@
 // Type Defines
 @class SKStateChart;
 typedef void (^StateBlock)(void);
-typedef void (^MessageBlock)(SKStateChart *sc);
\ No newline at end of file
+typedef void (^MessageBlock)(SKStateChart *sc);
+
+// Use this macro to simplify the final code
+// --- from ---
+// @"userPressedRedButton":^(SKStateChart *sc) {
+//     [sc goToState:@"red"];
+// },
+// --- to ---
+// SKDefineTransition_Event2State(@"userPressedRedButton", @"red"),
+
+#define SKDefineTransition_Event2State(event,nextState) event:^(SKStateChart *sc) { [sc goToState:nextState]; }
```

Comment:
Unbridgeable from swift, but I guess that's ok. Swift code can still use the slightly more verbose mapping right?